Marlin/ultralcd.cpp

```diff
@@ -761,10 +806,19 @@ static void lcd_move_menu_axis() {
   MENU_ITEM(back, MSG_MOVE_AXIS, lcd_move_menu);
   MENU_ITEM(submenu, MSG_MOVE_X, lcd_move_x);
   MENU_ITEM(submenu, MSG_MOVE_Y, lcd_move_y);
-  if (move_menu_scale < 10.0) {
+//if (move_menu_scale < 10.0) { //Why exclude Z and E from the 10mm and 100mm menus ???
```

Comment:
I don't know why this has been this way historically, but please submit this change separately and get some feedback first.